GPU-SUPPORT.md

```diff
@@ -0,0 +1,19 @@
+# GPU Support in Livepeer
+
+Livepeer enables node operators to transcode video on GPUs while concurrently mining cryptocurrencies. As there is a very wide range of GPU hardware out there, this document aims to crowdsource a list of specific GPU models that have been tested and verified to work on Livepeer. If you've tested an additional model, please submit an update to this document.
```

Comment:
Would it be good to extend this to a broader audience by saying "mining crypto currencies or other CUDA operations (e.g. rendering, Machine learning)" ?